### PR TITLE
Grammatical change

### PR DESCRIPTION
**Pull request by erolaliyev**
Replace this sentence with a brief description of the awesome changes your PR has. 😊

---

## ✅️ By submitting this PR, I have verified the following

- [] Added descriptive name to PR
  - Your PR should NOT be called `Update index.md` since it does not help the maintainer understand what has been changed.
  - For example, if you create a **Variables** article inside the **Python** directory, the pull request title should be **Python: add Variables article**.
  - Other examples are **Git: edit Git Commit article** or **PHP: create PHP section and add Data Structures article**
- [] Reviewed necessary formatting guidelines in [`CONTRIBUTING.md`](https://github.com/freeCodeCamp/guides/.github/CONTRIBUTING.md).
- [] No plagiarized, duplicate or repetitive  content that has been directly copied from another source.

### CODE REVIEW

**Comment by erolaliyev** [comment]
@Hijerboa I could not change description text without making any changes, so I added one semicolon and one comma in the appropriate places and updated description.

File: src/pages/javascript/es6/arrow_functions/index.md [hunk 1]
```diff
@@ -37,4 +37,4 @@ let NewOneWithParameters = (a, b) => {
 NewOneWithParameters(10, 20);
 ```
 
-I don’t think I need to give explanation for this. It's straightforward.
+I don’t think, I need to give explanation for this. It's straightforward.
```
Comment (Asjas) [inline src/pages/javascript/es6/arrow_functions/index.md]: Wouldn't the correct grammar be "I don't think I need to give **an** explanation for this"?

Comment (erolaliyev) [reply src/pages/javascript/es6/arrow_functions/index.md]: Thank you. That is the correct version.

**Comment by Hijerboa** [comment]
LGTM! Thank you for your contribution!

**Pull request merged**